app/views/notifications_mailer/pre_auction_report.text.erb

```diff
@@ -0,0 +1,24 @@
+Welcome to Bidtastic!
+
+Pre Auction Report:
+
+  +------+--------------+----------+---------+---------------+
+  | Name | Description  | Category | Min Bid | Bid Increment |
+  +------+--------------+----------+---------+---------------+
+
+  <%- @items.each do |item| %>
+    NAME: <%= item.name %>
+    DESCRIPTION: <%= truncate(item.description, length: 30, seperator: ' ') %>
+    CATEGORY <%= item.category %>
+    MIN BID: <%= item.minimum_bid %>
+    BID INCREMENT: <%= item.bid_increment %>
+
+  <% end %>
+
+table = TTY::Table.new header: ['Name','Description','Category','Min Bid','Bid Increment'],
```

Comment:
I'm not sure this works the way you want it to. Lets get it working or remove it.